### ISSUE TITLE
how to set constants - for a heavy tailed distribution. 

### ISSUE THREAD

**Issue by andrewcz** [NeXAIS/automean#1]
Hi @statsle - really interested in the paper.
I am just curious for in the limitations you said you had to choose constants for the algorithm.
But the algorithm is self tuning? 
so which constants do you need to set and how would you set them.

Thank you in advance 
Andrew

**Comment by qsunstats** [comment]
> Hi @statsle - really interested in the paper. I am just curious for in the limitations you said you had to choose constants for the algorithm. But the algorithm is self tuning? so which constants do you need to set and how would you set them.
> 
> Thank you in advance Andrew

Nah, the constants actually refer to the constants for nonasymptotic confidence intervals, which is not really discussed in the paper. I realized this could be misleading and thus removed this comment.
